sdk/schemaregistry/schema-registry/test/public/schemaRegistry.spec.ts

```diff
@@ -64,8 +64,9 @@ describe("SchemaRegistryClient", function () {
   let client: SchemaRegistryClient;
   let schema: SchemaDescription;
 
-  beforeEach(function (this: Context) {
-    ({ client, recorder } = createRecordedClient(this));
+  beforeEach(async function (this: Context) {
+    ({ client, recorder } = createRecordedClient(this.currentTest));
+    if (!env.SCHEMA_REGISTRY_GROUP) throw new Error("SCHEMA_REGISTRY_GROUP is not defined");
```

Comment:
should not the recorder already be loading this variable for us? why do we need this check?

Reply:
env doesn't know yet, `Record<string, string | undefined>` is the type.